Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/views/detail/AgendaDetailFragment.kt

```diff
@@ -246,6 +247,21 @@ class AgendaDetailFragment : Fragment() {
                 scheduleDetail.listRow.talkDescription.getHtmlFormattedSpanned()
 
         tv_agenda_detail_description.movementMethod = LinkMovementMethod.getInstance()
+        tv_agenda_detail_shareText.text = resources.getString(R.string.sharing_twitter_title)
+        imgv_twitter.setOnClickListener {
+            val twitter = viewModel.getSpeaker(scheduleDetail.listRow.primarySpeakerName)?.socialProfiles?.get("twitter")
+            var twitterVal = ""
+            if(twitter.isNullOrEmpty())
+                twitterVal = scheduleDetail.listRow.primarySpeakerName
+            else
+                twitterVal = "@"+twitter
```

Comment:
i included this in the suggestion above, but we could use a string template here like: `"@$twitter"`